command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
Why is the runtime client specification harmed by using more descriptive terms?  When the Solaris config was put together no one stated that all OCI runtimes must include support for solaris configs for testing purposes. This as you have stated for the CLI spec. Your insistence on using MUST language forces the reader to ask, ok who MUST. You are currently saying the runtime MUST. And you are indicating through these discussion that the runtime MUST provide a CLI for testing purposes. I do not believe the document will be harmed by being at tad more explicit.